src/s3000_laser.cc

```diff
@@ -62,30 +59,28 @@ class s3000node {
   string frameid_;
 
   double desired_freq_;
-  diagnostic_updater::FrequencyStatus freq_diag_;
 
+  typedef diagnostic_updater::DiagnosedPublisher<sensor_msgs::LaserScan> LaserScanDiagnosedPublisher;
+  boost::scoped_ptr<LaserScanDiagnosedPublisher> data_pub_;
 
-  s3000node(ros::NodeHandle h) : self_test_(), diagnostic_(),
+  s3000node(ros::NodeHandle h) :
     node_handle_(h), private_node_handle_("~"),
-    desired_freq_(20),
+    desired_freq_(16.7),
```

Comment:
In this other part of the code `16.6` is used:
https://github.com/clearpathrobotics/s3000_laser/pull/5/files#diff-3a47006afa665d86bf66c296076b69ebR120